vdr/vdr.go

```diff
@@ -158,7 +157,7 @@ func (r VDR) Update(id did.DID, current hash.SHA256Hash, next did.Document, _ *t
 	}
 
 	keyID := controllers[0].Authentication[0].ID.String()
-	_, err = r.network.CreateTransaction(didDocumentType, payload, keyID, nil, time.Now(), dag.TimelineIDField(meta.TimelineID), dag.TimelineVersionField(meta.Version+1))
+	_, err = r.network.CreateTransaction(didDocumentType, payload, keyID, nil, time.Now())
```

Comment:
`prev`-ing the previous version(s) comes in a future PR?

Reply:
yes, not part of the spec yet